proposed/http-middleware/middleware-meta.md

```diff
@@ -101,28 +110,23 @@ request and response being passed to the middleware.
 
 The primary downside of this interface is that the while the interface itself is
 a callable, there is currently no way to type hint a closure in a similar way.
-This may be resolved in the future with [functional interfaces][php-functional].
-
-[php-functional]: https://wiki.php.net/rfc/functional-interfaces
 
 ### 4.2 Single Pass (Lambda)
 
 The other approach to middleware is much closer to [StackPHP][stackphp] style
 and is defined as:
 
-
 ```
-fn(request, next): response
+fn(request, frame): response
 ```
 
 Middleware taking this approach generally has the following commonalities:
 
-* The middleware is defined as a [callable][php-callable] using a [closure][php-closure]
```

Comment:
As `[closure][php-closure]` is removed, `[php-closure]: http://php.net/closure` should be removed as well.